ioctl/newcmd/account/accountdelete_test.go

```diff
@@ -24,14 +28,21 @@ func TestNewAccountDelete(t *testing.T) {
 	client.EXPECT().SelectTranslation(gomock.Any()).Return("mockTranslationString",
 		config.English).AnyTimes()
 
-	client.EXPECT().GetAddress(gomock.Any()).Return("io1kuz56knh9g7x3ht36t2pgfd8fz5g6rxrh5a8dn", nil)
-
-	ks := keystore.NewKeyStore(config.ReadConfig.Wallet,
+	ks := keystore.NewKeyStore(os.TempDir(),
```

Comment:
i think better to create a new folder inside the tempDir, and delete this folder after test, something like:
```
testDir = NewDir(os.TempDir())
defer func() {
    os.RemoveDir(testDir)
}
```